tagstudio/tests/src/utils/test_str.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from parameterized import parameterized
+
+from tagstudio.src.core.utils.str import strip_punctuation
```

Comment:
`Tagstudio.src...` import format breaks at runtime for vscode and when running from the terminal

```suggestion
from src.core.utils.str import strip_punctuation
```

Reply:
Thank you for the feedback! I have made the changes.